bndtools.builder/src/org/bndtools/builder/BndtoolsBuilder.java

```diff
@@ -311,6 +313,8 @@
     protected void clean(IProgressMonitor monitor) throws CoreException {
         try {
             IProject myProject = getProject();
+
+            BuilderPlugin.clearComponentMarkers(myProject);
```

Comment:
Could we make ComponentMarker handle everything related to componentmarkers?  So that one class would both update and clear component markers?